Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -32,6 +32,8 @@ Settings can be specified at the global level to apply to the cluster as a whole
   Tags also exist that would give the latest version, but they are only recommended for test environments. For example, the tag `v17` will be updated each time a new Quincy build is released.
   Using the `v17` tag is not recommended in production because it may lead to inconsistent versions of the image running across different nodes in the cluster.
   * `allowUnsupported`: If `true`, allow an unsupported major version of the Ceph release. Currently `pacific` and `quincy` are supported. Future versions such as `reef` (v18) would require this to be set to `true`. Should be set to `false` in production.
+  `imagePullPolicy`: The image pull policy for the ceph daemon pods. Possible values are `Always`, `IfNotPresent`, and `Never`.
```

Comment:
just a small question, what will happen if mentioned `Never`, And image is not present

Reply:
pod startup will fail.